<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,20">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="QLabel" name="welcomeLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>7</y>
          <width>75</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Xin chào, ...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="logoutButton">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>10</y>
          <width>76</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Đăng xuất</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,17">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QListWidget" name="navWidget"/>
       </item>
       <item>
        <widget class="QStackedWidget" name="contentStackedWidget">
         <widget class="QWidget" name="page"/>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionGoodsManagement">
   <property name="text">
    <string>Quản lý Hàng Hóa</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
